website/docs/r/external_network_v2.html.markdown

```diff
@@ -14,11 +14,30 @@ Provides a VMware Cloud Director External Network resource (version 2). New vers
 uses new VCD API and is capable of creating NSX-T backed external networks as well as port group
 backed ones.
 
--> **Note:** This resource uses new VMware Cloud Director
-[OpenAPI](https://code.vmware.com/docs/11982/getting-started-with-vmware-cloud-director-openapi) and
-requires at least VCD *10.0+*. It supports both NSX-T and NSX-V backed networks (NSX-T *3.0+* requires VCD *10.1.1+*)
+-> This resource manages both NSX-T and NSX-V **External Networks** and **NSX-T Provider
```

Comment:
"both" refers to *two* elements, but we are listing three here
```suggestion
-> This resource manages NSX-T **External Networks**, NSX-V **External Networks**, and **NSX-T Provider**
```

Reply:
Very good catch - fixed